content/en/docs/instrumentation/erlang/sampling.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Sampling
+weight: 37
+---
+
+## Sampling
+
+[Sampling](/docs/concepts/sampling/) is a process that restricts the amount of
+traces that are generated by a system. The Erlang SDK offers several
+[head samplers](/docs/concepts/sampling#head-sampling).
+
+### Default behavior
+
+By default, all spans are sampled, and thus, 100% of traces are sampled. If you
+do not need to manage data volume, don't bother setting a sampler.
+
+### ParentBasedSampler
+
+When sampling, the `ParentBasedSampler` is most often used for
+[head sampler](/docs/concepts/sampling/#head-sampling). It uses the sampling
+decision of the Span's parent, or the fact that there is no parent, to know
+which secondary sampler to use.
+
+The sampler can be configured with environment variables `OTEL_TRACES_SAMPLER`
+and `OTEL_TRACES_SAMPLER_ARG` or using the Application configuration allows you
+to configure each of the 5 potential states of a Span's parent:
+
+- `root` - No parent
+- `remote_parent_sampled` - Parent is from a remote Span that is sampled
+- `remote_parent_not_sampled` - Parent is from a remote Span that is not sampled
+- `local_parent_sampled` - Parent is from a local Span that is sampled
+- `local_parent_not_sampled` - Parent is from a local Span that is not sampled
+
+#### TraceIdRatioBasedSampler
+
+Within the `ParentBasedSampler` the most common is the
+`TraceIdRatioBasedSampler`. It deterministically samples a percentage of traces
+that you pass in as a parameter.
```

Comment:
So this came up in the python PR, but I'm not really sure what the right recommendation is.

One gotcha with setting parentbased+traceidratio is that if you don't control an upstream service, your sampling rate isn't actually respected.

And the inverse gotcha is that if you have multiple services all using traceidratio and they set different sampling rates, you'll get turnt too.

Got an inkling towards a preferred setting + guidance? We'd need to be uniform across all languages, so I wanna make sure it's a coherent bit of guidance for everyone.

Reply:
Hm, yea, maybe needs some guidance around setting up sampling when dealing with a "public" endpoint.

But even in that case I think `ParentBased` with `TraceIdRatioBased` is the way to go, its just that you might want to configure `remote_parent_sampled` to also be `TraceIdRatioBased` instead of `AlwaysOn`.